mixer/test/client/env/setup.go

```diff
@@ -34,14 +34,15 @@ type TestSetup struct {
 	mfConf *MixerFilterConf
 	ports  *Ports
 
-	envoy         *Envoy
-	mixer         *MixerServer
-	backend       *HTTPServer
-	testName      uint16
-	stress        bool
-	faultInject   bool
-	noMixer       bool
-	mfConfVersion string
+	envoy    *Envoy
+	mixer    *MixerServer
+	backend  *HTTPServer
+	testName uint16
+	stress   bool
+	//The configurations of the filters before the Mixer filter
```

Comment:
remove the comment so all fields can be nicely aligned.

Reply:
Done.